File: webapi/DAL/Models/Bank.cs
using System;
using System.Collections.Generic;

namespace DAL.Models
{
    public partial class Bank
    {
        public Bank()
        {
            PaymentReceiverBanks = new HashSet<Payment>();
            PaymentSenderBanks = new HashSet<Payment>();
        }

        public int Id { get; set; }
        public string Name { get; set; } = null!;
        public string? Branch { get; set; }
        public string? Phone { get; set; }
        public string? Fax { get; set; }
        public string? PoBox { get; set; }
        public string? Website { get; set; }
        public string? Email { get; set; }

        public virtual ICollection<Payment> PaymentReceiverBanks { get; set; }
        public virtual ICollection<Payment> PaymentSenderBanks { get; set; }
    }
}

File: angular16-boilerplate/src/app/shared/models/bank.ts
export class Bank {
  public id: number;
  public name: string;
  public branch?: string | null;
  public phone?: string | null;
  public fax?: string | null;
  public poBox?: string | null;
  public website?: string | null;
  public email?: string | null;

  constructor(
    id: number,
    name: string,
    branch?: string | null,
    phone?: string | null,
    fax?: string | null,
    poBox?: string | null,
    website?: string | null,
    email?: string | null
  ) {
    this.id = id;
    this.name = name;
    this.branch = branch;
    this.phone = phone;
    this.fax = fax;
    this.poBox = poBox;
    this.website = website;
    this.email = email;
  }
}